protocol/communities/community_event.go

```diff
@@ -298,35 +298,29 @@ func (o *Community) updateCommunityDescriptionByCommunityEvent(communityEvent Co
 		}
 
 	case protobuf.CommunityEvent_COMMUNITY_MEMBER_KICK:
-		pk, err := common.HexToPubkey(communityEvent.MemberToAction)
-		if err != nil {
-			return err
-		}
-
-		if !o.IsControlNode() && o.IsPrivilegedMember(pk) {
-			return errors.New("attempt to kick an control node or privileged user from non-control node side")
```

Comment:
here or in validate we should check that someone didn't create a vulnerable admin event when we're kicking the owner

Reply:
I was about to add those checks in `validateAndGetEventsMessageCommunityDescription`. But next discovered there is already `validateCommunityEvent`. @mprakhov @osmaczko should not we somehow generalise it?

Reply:
It could be done as a part of [validateAndFilterEvents](https://github.com/status-im/status-go/blob/6bcf5f1289f9160168574290cbd6f90dede3f8f6/protocol/communities/manager.go#L1371), more precisely there: https://github.com/status-im/status-go/blob/6bcf5f1289f9160168574290cbd6f90dede3f8f6/protocol/communities/community.go#L2148

Reply:
done